WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -299,6 +300,7 @@ def time_until_competition
 
   def show
     @competition = competition_from_params
+    @nearby_competitions = get_nearby_competitions(@competition)
```

Comment:
Probably shouldn't be doing this query unless we're actually going to be using it.